docs/integrations/web-servers/opentelemetry/varnish-opentelemetry.md

```diff
@@ -0,0 +1,169 @@
+
+<img src={useBaseUrl('img/integrations/web-servers/varnish.png')} alt="Thumbnail icon" width="75"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for [Varnish](https://varnish-cache.org/intro/) provides dashboards that help you analyze logs generated by Varnish servers. This app allows you to identify traffic sources, improve application and website workflows, and understand how customers use your product.
+
+Varnish logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Varnish-OpenTelemetry/Varnish-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Varnish
+
+- **`webengine.cluster.name`** - User configured. Enter a name to identify this Varnish cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.system`**  - Has a fixed value of **varnish**.
+- **`sumo.datasource`** - Has a fixed value of **varnish**.
+
+## Prerequisites
+
+1.  Configure logging in Varnish. Varnish supports logging via local text log files. For details please visit this [page](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/). For the dashboards to work properly, please set the Varnish log format as explained [here](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/#step-3-customise-options-1):
+```
+%h %l %u %t \"%r\" %s %b \"%{Referer}i\" \"%{User-agent}i\"
+```
+
+1.  Configure Varnish to log to a local file. By default, any installation of varnishd will not write any request logs to disk. Instead, Varnish has an in-memory log, and supplies tools to tap into this log and write to disk. To configure logging to a local file, follow the steps on [this](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/#enable-varnishncsa-logging) page. By default, Varnish logs are stored in /var/log/varnish/varnishncsa.log. For customized options please visit this [page](https://docs.varnish-software.com/tutorials/enabling-logging-with-varnishncsa/#step-3-customise-options-1).
+
+## Collection Configuration & App installation
+
+As part of setting up the collection process and app installation user can select the App from App Catalog and click Install App. Please follow the steps below:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the "Use an existing Collector" option.
+:::
+
+If you want to use an existing OTel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
```

Comment:
This line can be deleted, as this is already available as part of the suggestion made above.